aws/templates/ml-endpoint.template

```diff
@@ -8,12 +8,15 @@ Metadata:
           default: "Interface Endpoint Configuration"
         Parameters:
           - S3Bucket
+          - VerDir
           - Vpc
           - SecurityGroup
           - Subnets
     ParameterLabels:
       S3Bucket:
         default: 'S3 Bucket Name'
+      VerDir:
```

Comment:
For this parameter `verDir`, I think it's better to name it `S3Directory`, since we already have a parameter called `S3Bucket`.